src/peft/tuners/lora/config.py

```diff
@@ -89,6 +93,18 @@ class LoraConfig(PeftConfig):
         metadata={"help": "Set this to True if the layer to replace stores weight like (fan_in, fan_out)"},
     )
     bias: str = field(default="none", metadata={"help": "Bias type for Lora. Can be 'none', 'all' or 'lora_only'"})
+    use_rslora: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "When set to True, uses "
+                "<a href='https://doi.org/10.48550/arXiv.2312.03732'>Rank-Stabilized LoRA</a> "
```

Comment:
Let's not use html syntax in the help.

Reply:
I interpreted this to mean that I should still include the url but without the html syntax.